### PR TITLE
Making contracts not upgradeable but migrable

### PR DESCRIPTION
**Pull request by sullof**
Changes:

- Park the Transparent Vault contracts, as they are not used in the current version of the protocol
- Introduce an airdroppable version of the vault, using ERC6551 to handle the assets
- Add in the lockable folder the ERC721Lockable contracts, taken from [ndujaLabs/erc721lockable](https://github.com/ndujalabs/erc721lockable) repo, for completeness
- Make the CrunaSafebox not upgradeable for improved trust, allowing migration to V2 in the future
- Make the Protected not upgradeable and able to have more than one protector
- Modified the OwnerNFT to separate the roles between owner and minters

### CODE REVIEW

File: test/SafeBox.test.js [hunk 1]
```diff
@@ -0,0 +1,249 @@
+const {expect} = require("chai");
+const {deployContractUpgradeable, deployContract, amount, assertThrowsMessage} = require("./helpers");
+const DeployUtils = require("../scripts/lib/DeployUtils");
+
+describe("SafeBox", function () {
+  const deployUtils = new DeployUtils(ethers);
+
+  let protected, safeBox;
```
Comment (andyscraven) [inline test/SafeBox.test.js]: protected is a reserved word and Webstorm is kicking off about it.  Perhaps "theProtected"

Comment (sullof) [reply test/SafeBox.test.js:8]: Rename it as you prefer

Comment (sullof) [reply test/SafeBox.test.js]: Renamed

**Review by andyscraven — APPROVED** [review]
This is too big a change to do it justice so I will put aside a day to write some tests, try to find edge cases that might break after we merge.
